games/lockpicking/src/index.html

```diff
@@ -10,10 +10,11 @@
       href="https://fonts.googleapis.com/css2?family=EB+Garamond:wght@500;800&display=swap"
       rel="stylesheet"
     />
-
     <link rel="stylesheet" href="./styles/app.css" />
     <link rel="stylesheet" href="./styles/pages/home.css" />
     <link rel="stylesheet" href="./styles/game/popup.css" />
+    <link rel="stylesheet" href="./styles/pages/score-tabs.css" />
+    <link rel="stylesheet" href="./styles/pages/tabs-table.css" />
```

Comment:
Not a tabs table. 
Should be something like scores-table or abstract table styles.